libr/parse/p/parse_x86_pseudo.c

```diff
@@ -107,9 +109,15 @@ static int replace(int argc, const char *argv[], char *newstr) {
 	for (i = 0; ops[i].op != NULL; i++) {
 		if (!strcmp (ops[i].op, argv[0])) {
 			if (newstr != NULL) {
+				d = 0;
 				for (j = k = 0; ops[i].str[j] != '\0'; j++, k++) {
-					if (ops[i].str[j] > '0' && ops[i].str[j] <= '9') {
-						const char *w = argv[ops[i].str[j] - '0'];
+					if (ops[i].str[j] == '#') {
+						if (ops[i].args[d] <= 0 || d >= kMaxOps) {
+							// XXX Shouldn't ever happen...
```

Comment:
Assign ops[i.str[j to a variable instead of dereferencing 2 pointers several times